src/test/java/uk/gov/hmcts/divorce/caseworker/service/notification/PersonalServiceNotificationTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.divorce.caseworker.service.notification;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+import uk.gov.hmcts.divorce.notification.NotificationService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.join;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.divorce.common.model.LanguagePreference.ENGLISH;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.APPLICANT_SOLICITOR_PERSONAL_SERVICE;
+import static uk.gov.hmcts.divorce.notification.FormatUtil.formatId;
+import static uk.gov.hmcts.divorce.notification.NotificationConstants.APPLICANT_NAME;
+import static uk.gov.hmcts.divorce.notification.NotificationConstants.APPLICATION_REFERENCE;
+import static uk.gov.hmcts.divorce.notification.NotificationConstants.RESPONDENT_NAME;
+import static uk.gov.hmcts.divorce.notification.NotificationConstants.SOLICITOR_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.APPLICANT_2_FIRST_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_CASE_ID;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_FIRST_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_LAST_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_SOLICITOR_EMAIL;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_SOLICITOR_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestDataHelper.applicantRepresentedBySolicitor;
+
+@ExtendWith(MockitoExtension.class)
+class PersonalServiceNotificationTest {
+
+    @Mock
+    private NotificationService notificationService;
+
+    @Mock
+    private CommonContent commonContent;
+
+    @InjectMocks
+    private PersonalServiceNotification personalServiceNotification;
+
+    @Test
+    void shouldSendPersonalServiceNotificationToApplicantSolicitor() {
```

Comment:
would be nice to add when part so `shouldSendPersonalServiceNotificationToApplicantSolicitorWhenServiceMethodIsPersonal` or something similar

Reply:
Not so sure about this, as the PersonalServiceNotification component does not actually make the decision to send or not to send according to the service method.  I let the controller in the event decide.